android/build.gradle

```diff
@@ -36,8 +36,8 @@ allprojects {
 }
 
 dependencies {
-    compile 'com.couchbase.lite:couchbase-lite-android:1.3.1'
-    compile 'com.couchbase.lite:couchbase-lite-java-listener:1.3.1'
-    compile 'com.couchbase.lite:couchbase-lite-java-javascript:1.3.1'
-    compile 'com.facebook.react:react-native:0.14.+'
+    compile 'com.couchbase.lite:couchbase-lite-android:1.4.1’
+    compile 'com.couchbase.lite:couchbase-lite-java-listener:1.4.1’
+    compile 'com.couchbase.lite:couchbase-lite-java-javascript:1.4.1’
+    compile ‘com.facebook.react:react-native:0.14.+’
```

Comment:
should the single quotes on line 42 be `'`?

Reply:
yes... i changed it